installer/vanilla.porter.yaml

```
@@ -68,6 +68,13 @@ parameters:
     default: "13.6.2"
     applyTo:
       - install
+  - name: operatorNamespace
+    description: Namespace to install the operator into
+    type: string
+    default: "porter-operator-system"
+    applyTo:
+      - install
+      - upgrade
```

The `configureNamespace` custom action will also need the porter operator namespace parameter. 
```suggestion
    applyTo:
      - install
      - upgrade
      - configureNamespace
```